### DISCUSSION TITLE
One click install

### DISCUSSION THREAD

**Discussion by rafa-acioly** [vitodeploy/vito#404] [Ideas]
Some apps use third party services such as MeiliSearch, Docuseal, MinIO, PostHog and etc, it would be awesome if we could install services in one-click.
Enabling this feature would also open the possibility to other devs to include more services.

**Comment by saeedvaziry** [comment]
I recommend dropping an idea discussion for every service in the list. For now I would only start with MeiliSearch.

Some other services like a mail server are also in the list to be added.

**Reply by rafa-acioly** [reply]
Great, here's some ideas.

MeiliSearch - search engine
MinIO - S3 compatible object storage
PostHog - Product analytics, feature flags, session replays, experiments and surveys
Metabase - Insight dashboard based on real data
Plausible - Google Analytics alternative

**Comment by saeedvaziry** [comment]
Now we can use plugins to add those services and no need to add them to the core of vito
